### PR TITLE
docs(2020-11-25): add usage analytics app pr review notes

### PR DESCRIPTION
**Pull request by ismay**
I was working on a google doc about the meeting, but then I thought why not submit it to notes as meeting notes. Let me know if you all agree or if I should change anything!

### CODE REVIEW

**Review by Mohammer5 — APPROVED** [review]

**Review by varl — APPROVED** [review]
Consistent with my expectations after the meeting, nice summary!

File: decisions/2020/11/25-usage-analytics-app-review.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- Use `__test__` folders instead of co locating tests with the tested files.
+- Update all dependencies (which would also address the cli style import sorting).
+- See if the state variables in the App.js component could be simplified a bit. There are quite a lot of them currently.
+- Parameter destructuring, should we destructure so it’s explicit, or pass params without destructuring for brevity (no conclusion yet)?
+- Some idiosyncracies that ended up in the refactor could probably have been avoided if multiple people had been involved from the start, like with the sms app refactor. Not much we can do about that now, but good to keep in mind for the future.
+- Fix the responsive width for the chart (currently doesn't downsize)
+
+### Address in the PR
+
+- Rename interval and setInterval (so we don’t shadow the global setInterval).
```
Comment (amcgee) [inline decisions/2020/11/25-usage-analytics-app-review.md:24]: I think this is a typo?  The first `setInterval` should be something else

**Review by amcgee — APPROVED** [review]
LGTM, one minor typo

File: decisions/2020/11/25-usage-analytics-app-review.md (hunk 1 above)
Comment (ismay) [reply decisions/2020/11/25-usage-analytics-app-review.md:24]: What I mean there is it's currently called setInterval, and we should change it to something else. (I renamed it to setReportInterval).

Comment (ismay) [reply decisions/2020/11/25-usage-analytics-app-review.md:24]: But should be ok as it is currently right? Or I'm overlooking something.

Comment (amcgee) [reply decisions/2020/11/25-usage-analytics-app-review.md:24]: oh yeah, my bad - I read **to** instead of **and**, all clear

**Pull request merged**